src/services/school/hooks/index.js

```diff
@@ -71,7 +71,7 @@ const setStudentsCanCreateTeams = async (context) => {
 				throw new Error('method not supported');
 		}
 	} catch (error) {
-		logger.error(error);
+		throw new Error('error while determing school settings for team creation by user');
```

Comment:
Error replace with feather errors and pass the error as second parameter.

Reply:
And it is implemented as an non blocking hook, throw it up will change the logic. Your are sure that it work?
But i see some more hooks that we can improve. 
This can mark as part from https://ticketsystem.hpi-schul-cloud.org/browse/SC-7188

Reply:
ok, than lets to this correct and move it to 7188?

Reply:
@CeEv I propose to close this one and do a solid cleanup in SC-7188. Ok for you?